src/shell/autoexec.cpp

```diff
@@ -509,9 +519,21 @@ void AutoExecModule::AutoMountDrive(const std::string& dir_letter)
 	}
 }
 
-void AutoExecModule::AddLine(const Placement placement, const std::string& line)
+void AutoExecModule::AutoMountDriveC(const std::string& directory)
 {
-	autoexec_lines[placement].push_back(line);
+	constexpr auto placement = Placement::InitialAutogeneratedCommands;
+	AddLine(placement, CmdMount + std::string("-u C") + ToNul);
+	AddLine(placement, CmdMount + std::string("C ") + directory + ToNul);
```

Comment:
std::string can add literals, so maybe a simplification to avoid the `std::string(...)` cast/temporary.

Reply:
Ok, changed.